content/modals/GB/short_term_xo.json

```
@@ -0,0 +1,51 @@
+{
+    "meta": {
+        "product": "PAY_LATER_SHORT_TERM",
+        "periodicPayment": "{formattedPeriodicPayment}",
+        "minAmount": "{minAmount}",
+        "maxAmount": "{maxAmount}",
+        "qualifying": "{qualifying_offer}",
+        "amount": "{transaction_amount}",
+        "apr": "{apr}",
+        "preapproved": true,
+        "variables": {
+            "transaction_amount": "${eval(transaction_amount ? transaction_amount : '-')}",
+            "qualifying_offer": "${eval(CREDIT_OFFERS_DS.qualifying_offer ? CREDIT_OFFERS_DS.qualifying_offer : 'false')}",
+            "financing_code": "${CREDIT_OFFERS_DS.financing_code}",
+            "formattedPeriodicPayment": "${CREDIT_OFFERS_DS.formattedPeriodicPayment}",
+            "total_payments": "${CREDIT_OFFERS_DS.total_payments}",
+            "formattedMinAmount": "${CREDIT_OFFERS_DS.formattedMinAmount}",
+            "formattedMaxAmount": "${CREDIT_OFFERS_DS.formattedMaxAmount}",
+            "formattedTotalCost": "${CREDIT_OFFERS_DS.formattedTotalCost}",
+            "minAmount": "${CREDIT_OFFERS_DS.minAmount}",
+            "maxAmount": "${CREDIT_OFFERS_DS.maxAmount}",
+            "apr": "${CREDIT_OFFERS_DS.apr}",
+            "nominal_rate": "${CREDIT_OFFERS_DS.nominal_rate}"
+        }
+    },
+    "content": {
+        "headline": "Pay in 3 interest-free payments",
+        "subheadline": "Available for purchases of {formattedMinAmount} to {formattedMaxAmount}. No sign-up fees or late fees.",
+        "qualifyingSubheadline": "Split your purchase of {formattedTotalCost} into {total_payments} with no sign-up fees or late fees.",
+        "donutTimestamps": ["Today", "1 month", "2 months"],
+        "estimatedInstallments": {
+            "items": [
+                { "total_payment:": "{formattedPeriodicPayment}", "payment_date": "Today" },
+                { "total_payment:": "{formattedPeriodicPayment}", "payment_date": "1 month" },
+                { "total_payment:": "{formattedPeriodicPayment}", "payment_date": "2 months" }
+            ]
+        },
+        "instructions": {
+            "instructionHeadline": "How to use Pay in 3",
+            "instructionSubHeadline": "Confirm your info and apply, set up autopay, and review your purchase."
+        },
+        "preapproval": {
+            "preapprovalHeadline": "Pre-approved to Pay in 3 on this purchase ",
+            "preapprovalSubHeadline": "Pay {formattedTotalCost} in {total_payments} interest-free payments.",
+            "preapprovalDisclaimerHeadline": "What to know about pre-approval",
+            "preapprovalDisclaimerBody": "Pre-approval may not be valid if you make another Pay in 3 purchase before the session expires."
+        },
```

```suggestion
        },
        "cta": {
            "buttonTextEligible": "Continue with Pay in 3",
            "buttonTextIneligible": "Back to Checkout",
            "link": "https://www.paypal.com/link/to/short-term-application"
        },
```
Looks like we're missing the call to action checkout button content similar to what we have in our Pay Monthly checkout modal. We can add it right below the preapproval content.